src/main/drivers/sdio_h7xx.c

```diff
@@ -168,8 +167,14 @@ void HAL_SD_MspInit(SD_HandleTypeDef* hsd)
     }
 
     if (sdioHardware->instance == SDMMC1) {
+        __HAL_RCC_SDMMC1_CLK_DISABLE();
+        __HAL_RCC_SDMMC1_FORCE_RESET();
+        __HAL_RCC_SDMMC1_RELEASE_RESET();
         __HAL_RCC_SDMMC1_CLK_ENABLE();
     } else if (sdioHardware->instance == SDMMC2) {
+        __HAL_RCC_SDMMC2_CLK_DISABLE();
+        __HAL_RCC_SDMMC2_FORCE_RESET();
+        __HAL_RCC_SDMMC1_RELEASE_RESET();
```

Comment:
Should this be `__HAL_RCC_SDMMC2_RELEASE_RESET();`?

Reply:
yes, well spotted and thanks for the review.  much appreciated!  fixed.